.xperior/testds/motr-single_tests.yaml

```diff
@@ -20,7 +20,1267 @@
 ---
 Tests:
   - id       : 00userspace-tests
-    script   : 'm0 run-ut'
+    script   : 'm0 run-ut -t libm0-ut'
+    dir      : src/scripts
+    executor : Xperior::Executor::MotrTest
+    #executor : Xperior::Executor::Skip
+    sandbox  : /var/motr/m0ut
+    groupname: 01motr-single-node
+    polltime : 15
+    timeout  : 2400
+
+  - id       : 00userspace-tests
```

Comment:
Use unique id for each test to avoid overwriting the test-result in the final report.
For example : you can append the UT name to `00userspace-tests` to get a unique id as `00userspace-tests-libm0-ut`

Reply:
done